app/gcl/agwa/agwa.css

```
@@ -0,0 +1,557 @@
+    --accent-light: #e8f5ee;
+    --accent2: #1d5fa8;
+    --accent2-light: #e8f0fb;
+    --warn: #b85c00;
+    --warn-light: #fef3e8;
+    --tip: #5c3fa8;
+    --tip-light: #f2eeff;
+    --border: #e2dfd8;
+    --border2: #ccc9c2;
+    --tag-bg: #f0ede6;
+    --sidebar-w: 280px;
+    --font-display: 'DM Serif Display', Georgia, serif;
```

Stylelint (17.11.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)